SleepEngine/Engine/Jobs/Job.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <Engine/Jobs/JobAffinity.h>
+
+BEGIN_NAMESPACE_SLEEP
+
+class Job
+{
+public:
+    Job(JobAffinity affinity = JobAffinity::Generic)
+        : m_affinity(affinity) {}

```

Comment:
Why you default initialize affinity here?

Reply:
probably for testing, I ll remove it